packages/matchbox/src/components/Layout/Section.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Column } from '../Column';
+
+type SectionProps = {
+  children?: React.ReactNode;
+  annotated?: boolean;
+  'data-id'?: string;
+};
+
+function Section(props) {
```

Comment:
i think the types need to be assigned on the function parameter in order for end users to pick up on the types